types/oauth2-server/index.d.ts

```diff
@@ -256,7 +256,7 @@ declare namespace OAuth2Server {
          * @param accessToken
          * @param callback
          */
-        getAccessToken(accessToken: string, callback?: Callback<Token>): Promise<Token>;
+        getAccessToken(accessToken: string, callback?: Callback<Token>): Promise<Token | Falsey>;
```

Comment:
I'm not sure this is the case. If you return a falsey value then validateAccessToken and validateScope could fail because they expect valid access tokens. I think it's better to throw an exception if you can't retrieve the access token. It won't be as obvious when you get a "accessToken" not a member of undefined error later down the chain.

Reply:
Unless you are using strictNullCheck in tsconfig, I fail to see how the dev benefits from this change. By default, in this instance, the return type can be Token, null or undefined. Explicity adding falsey requires the dev to do a type check on the result of getAccessToken to determine if it's a Token or Falsey.

Reply:
This is just a type definition -- if the actual library returns `false` then the types should reflect that, right?